domains/api.anshtyagi.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "API WEBSITE",
+    "repo": "https://github.com/anshtyagi0",
+    "owner": {
+        "username": "anshtyagi0",
+        "email": "tyagiansh175@gmail.com"
+    },
+    "record": {
+        "CNAME": "85fea144-dc5a-45d3-84e3-c3400bf93e87.id.repl.co",
```

```suggestion
        "CNAME": "85fea144-dc5a-45d3-84e3-c3400bf93e87.id.repl.co"
```
Trailing commas are not supported in JSON.